openedx/core/djangoapps/video_pipeline/config/waffle.py

```diff
@@ -10,15 +10,14 @@
 WAFFLE_NAMESPACE = 'videos'
 
 # .. toggle_name: videos.deprecate_youtube
-# .. toggle_implementation: WaffleFlag
+# .. toggle_implementation: CourseWaffleFlag
```

Comment:
**Note**: Linting may be able to catch some of these issues. There is at least one more case in this PR.

This comment is a note for future improvements.